docs/ci.md

```diff
@@ -2,15 +2,21 @@
 
 ## Unit tests
 
-All unit tests currently live in the `tests/` directory.
+Unit tests are designed to test specific InstructLab components or features in isolation. Generally, new code should be adding or modifying unit tests.
+
+All unit tests currently live in the `tests/` directory and are run with [pytest](https://docs.pytest.org/) via [tox](https://tox.wiki/).
 
 ## Functional tests
 
-The functional test script can be found at `scripts/functional-tests.sh`
+Functional tests are designed to test InstructLab components or features in tandem, but not necessarily as part of a complex workflow. New code may or may not need a functional test but should strive to implement one if possible.
```

Comment:
The line between adding to functional tests and e2e tests is still a bit fuzzy.  Part of the reason for the distinction is also a matter of what can run on the target hardware right?

Reply:
Good point - I'll modify the text!